sdk/keyvault/azure-security-keyvault-keys/test/ut/mocked_transport_adapter_test.hpp

```diff
@@ -90,14 +90,14 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys { nam
       {
         keyType = "oct";
       }
-
-      result = new char[std::string(fakeKey).size() + keyType.size()];
+      size_t bufferSize = std::string(fakeKey).size() + keyType.size();
+      result = new char[bufferSize];
 
 #if defined(_MSC_VER)
 #pragma warning(push)
 #pragma warning(disable : 4996)
 #endif
```

Comment:
Given that now we use `snprintf()` instead of `sprintf()`, this warning suppression is no longer needed?

Reply:
done